<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>equipmentEditor</class>
 <widget class="QDialog" name="equipmentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equipment Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_filter">
         <property name="placeholderText">
          <string>Filter</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="equipmentsListView"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonAdd">
           <property name="toolTip">
            <string>Add equipment</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../brewtarget.qrc">
             <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonRemove">
           <property name="toolTip">
            <string>Remove equipment</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../brewtarget.qrc">
             <normaloff>:/images/smallMinus.svg</normaloff>:/images/smallMinus.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabEquipmentWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabEquipmentDetails">
        <attribute name="title">
         <string>Details</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="toolTip">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BtVolumeLabel" name="label_boilSize">
           <property name="text">
            <string>Pre-boil volume</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_boilSize">
           <property name="toolTip">
            <string>Pre-boil volume</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="BtVolumeLabel" name="label_batchSize">
           <property name="text">
            <string>Batch size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_batchSize">
           <property name="toolTip">
            <string>Batch size</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_calcBoilVolume">
           <property name="toolTip">
            <string>If checked, we will calculate your pre-boil volume based on your desired batch size, boil time, evaporation rate, losses, etc.</string>
           </property>
           <property name="text">
            <string>Calculate pre-boil volume</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_defaultEquipment">
           <property name="text">
            <string>Set as default</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEquipmentBoilingAndWater">
        <attribute name="title">
         <string>Boiling &amp;&amp; Water</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="BtTimeLabel" name="label_boilTime">
           <property name="text">
            <string>Boil time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="BtTimeEdit" name="lineEdit_boilTime">
           <property name="toolTip">
            <string>Boil time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BtVolumeLabel" name="label_evaporationRate">
           <property name="text">
            <string>Evaporation rate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_evaporationRate">
           <property name="toolTip">
            <string>How much water boils off per hour</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="BtVolumeLabel" name="label_topUpKettle">
           <property name="text">
            <string>Kettle top-up water</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_topUpKettle">
           <property name="toolTip">
            <string>How much water is added to kettle immediately pre-boil</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="BtVolumeLabel" name="label_topUpWater">
           <property name="text">
            <string>Final top-up water</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_topUpWater">
           <property name="toolTip">
            <string>Water added to fermenter</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="BtVolumeLabel" name="label_grainAbsorption">
           <property name="text">
            <string>Grain absorption</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="BtGenericEdit" name="lineEdit_grainAbsorption"/>
         </item>
         <item row="5" column="0">
          <widget class="BtTemperatureLabel" name="label_boilingPoint">
           <property name="text">
            <string>Boiling point of water</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="BtTemperatureEdit" name="lineEdit_boilingPoint">
           <property name="toolTip">
            <string>Temperature at which water boils</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_hopUtilization">
           <property name="text">
            <string>Hop utilization</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="BtGenericEdit" name="lineEdit_hopUtilization"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEquipmentMashTunDetails">
        <attribute name="title">
         <string>Mash Tun Details</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="BtVolumeLabel" name="label_tunVolume">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_tunVolume">
           <property name="toolTip">
            <string>Volume of mash tun</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BtMassLabel" name="label_tunWeight">
           <property name="text">
            <string>Mass</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="BtMassEdit" name="lineEdit_tunWeight">
           <property name="toolTip">
            <string>Mass or weight of mash tun</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_specificHeat">
           <property name="text">
            <string>Specfic Heat (cal/g*K))</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="BtGenericEdit" name="lineEdit_tunSpecificHeat">
           <property name="toolTip">
            <string>Mash tun material specific heat</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEquipmentLosses">
        <attribute name="title">
         <string>Losses</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="BtVolumeLabel" name="label_trubChillerLoss">
           <property name="text">
            <string>Kettle to fermenter</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_trubChillerLoss">
           <property name="toolTip">
            <string>Wort lost between kettle and fermenter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BtVolumeLabel" name="label_lauterDeadspace">
           <property name="text">
            <string>Lauter deadspace</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="BtVolumeEdit" name="lineEdit_lauterDeadspace">
           <property name="toolTip">
            <string>Volume of wort lost to lauter deadspace</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabEquipmentNotes">
        <attribute name="title">
         <string>Notes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEdit_notes">
           <property name="toolTip">
            <string/>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionRestoreDefaultValue">
   <property name="icon">
    <iconset resource="../brewtarget.qrc">
     <normaloff>:/images/restoreDefault.png</normaloff>:/images/restoreDefault.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Restore grain absorption default value</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtGenericEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>BtVolumeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTimeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTimeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtMassLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtMassEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../brewtarget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>equipmentEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>equipmentEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
